plugins/com.python.pydev.analysis/src/com/python/pydev/analysis/pylint/PyLintAnalysis.java

```diff
@@ -235,7 +239,7 @@ public void afterRunProcess(String output, String errors, IExternalCodeAnalysisS
     // \Z = end of the input
 
     private static Pattern PYLINT_MATCH_PATTERN = Pattern
-            .compile("\\A[CRWEF]:\\s*(\\d+)\\s*,\\s*(\\d+)\\s*:(.*)\\((.*)\\)\\s*\\Z");
+            .compile("\\A[CRWEFI]:\\s*(\\d+)\\s*,\\s*(\\d+)\\s*:\\s*\\((.*)\\)((.|\\n)*)\\s*\\Z");
```

Comment:
The logic inside of addToMarkers must be updated to consider the new format of the expressions (please create a test case for the `afterRunProcess()` method).